.pre-commit-config.yaml

```diff
@@ -74,7 +74,7 @@ repos:
         args:
           [
             "-c",
-            "cd ui-v2 && npm i --no-upgrade --silent --no-progress && npm run lint",
+            ". $NVM_DIR/nvm.sh && cd ui-v2 && nvm use && npm i --no-upgrade --silent --no-progress && npm run lint",
```

Comment:
the sourcing of `nvm.sh` is necessary because `pre-commit` runs commands in a non-interactive shell which means that `bashrc` and such files are not executed which is normally where `nvm` is activated